src/build/stream-resolver.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import * as fs from 'fs';
+import {Resolver, Deferred, FSResolver} from 'hydrolysis';
+import * as minimatch from 'minimatch';
+import * as path from 'path';
+import {Transform} from 'stream';
+import File = require('vinyl');
+import * as url from 'url';
+
+export interface Config {
+  entrypoint?: string;
+  /**
+   * Hostname to match for absolute urls.
+   * Matches "/" by default
+   */
+  host?: string;
+  /**
+   * Prefix directory for components in url. Defaults to "/".
```

Comment:
I'd like to see a default config instance that StreamResolver references, since it's really StreamResolver that has the defaults, not the Config interface.